roles/business.md

```diff
@@ -2,6 +2,22 @@
 
 ## Business Traits
 
+### +Academy
+
+We need someone:
+
+- to prepare and deliver the weekly learning programmes
+- to provide appropriate feedback and coaching, to help academy members to succeed  
+- to use a variety of teaching methods to challenge and engage academy members
+- to be the PO for the greenfield project
+
+We expect someone in the role:
+
+- to be autonomous in leading the academy
+- to be well organised and committed
```

Comment:
Yeah think the current wording is soggy, sounds like a basic role expectation

- to consistently organise and deliver valuable learnings for the academy members
- to be available in the office during core academy hours (x am - y pm)